populate/src/main/java/bio/terra/pearl/populate/service/EnrolleePopulator.java

```diff
@@ -430,7 +458,29 @@ public Enrollee createNew(EnrolleePopDto popDto, StudyPopulateContext context, b
         if (popDto.isWithdrawn()) {
             withdrawnEnrolleeService.withdrawEnrollee(enrollee);
         }
-        return enrollee;
+    }
+
+    private EnrolleePopulationEntities createNewGovernedEnrollee(EnrolleePopDto popDto, StudyPopulateContext context, ProxyPopDto proxyPopDto) {
+        StudyEnvironment studyEnvironment = studyEnvironmentService.find(popDto.getStudyEnvironmentId()).get();
```

Comment:
this method should take the StudyEnvrionment as an argument to avoid this query -- we already have it in the caller as `attachedEnv`